libmachine/cert/cert.go

```diff
@@ -195,12 +207,15 @@ func ValidateCertificate(addr, caCertPath, serverCertPath, serverKeyPath string)
 		return false, ErrValidatingCert{err}
 	}
 
+	log.Debugf("Configuration: +%v", tlsConfig)
+
 	dialer := &net.Dialer{
 		Timeout: time.Second * 2,
 	}
 
 	_, err = tls.DialWithDialer(dialer, "tcp", addr, tlsConfig)
 	if err != nil {
+		log.Debugf("Certs are not valid: %s", err)
```

Comment:
@carolynvs: Does your output refer to this line: https://github.com/docker/machine/issues/1954#issuecomment-150039974 ?